abb_irb1200_5_90_moveit_config/config/kinematics.yaml

```diff
@@ -1,5 +1,12 @@
 manipulator:
-  kinematics_solver: trac_ik_kinematics_plugin/TRAC_IKKinematicsPlugin
-  kinematics_solver_search_resolution: 0.005
-  kinematics_solver_timeout: 0.005
-  kinematics_solver_attempts: 3
\ No newline at end of file
+  kinematics_solver: moveit_opw_kinematics_plugin/MoveItOPWKinematicsPlugin
+  opw_kinematics_geometric_parameters:
+      a1:  0
+      a2: -0.042
+      b:   0.0
+      c1:  0.3991
+      c2:  0.448
+      c3:  0.451
+      c4:  0.082
+  opw_kinematics_joint_offsets: [0.0, 0.0, -1.57079632679, 0.0, 0.0, 0.0]
```

Comment:
Any reason not to use `deg(..)` here?

Reply:
No, thanks for the reminder, added a commit with `deg()` syntax.